ui/src/components/databaseComponents/asideComponents/AsideComponent.vue

```diff
@@ -19,335 +19,192 @@
+                                </template>
+                            </el-menu-item>
+                        </div>
                     </div>
                 </el-submenu>
             </div>
         </el-menu>
-        <div v-if="showRightMenu" class="right-menu border-radius-little box-shadow"
-            :style="{ top: topNumber + 'px', left: leftNumber + 'px' }">
-            <right-menu-component :rightMenuList="rightMenuList"></right-menu-component>
-        </div>
+        <el-dialog title="Tips" :visible.sync="dialogVisible" width="25%" center>
+            <span>Are you sure to delete this {{rightClickType}}?</span>
+            <span slot="footer" class="dialog-footer">
+                <el-button @click="dialogVisible = false">cancel</el-button>
+                <el-button type="primary" @click="deleteGroupOrFile">delete</el-button>
+            </span>
+        </el-dialog>
+        <el-dialog width="25%" center :title="`${setGroup} group`" :visible.sync="dialogGroupVisible">
+            <el-form ref="ruleForm" :rules="rules" :model="groupForm" label-position="left">
+                <el-form-item label="group name" label-width="100px" prop="name">
+                    <el-input :disabled="setGroup == 'edit'" v-model="groupForm.name" autocomplete="off"
+                        style="width: 300px;"></el-input>
+                </el-form-item>
+                <el-form-item label="group type" label-width="100px" prop="catalog">
+                    <el-select v-model="groupForm.catalog" style="width: 300px;" placeholder="please select">
+                        <el-option label="CATALOG_STREAM" value="CATALOG_STREAM"></el-option>
+                        <el-option label="CATALOG_MEASURE" value="CATALOG_MEASURE"></el-option>
+                    </el-select>
+                </el-form-item>
+            </el-form>
+            <div slot="footer" class="dialog-footer">
+                <el-button @click="dialogGroupVisible = false">cancel</el-button>
+                <el-button type="primary" @click="confirmForm">{{setGroup}}
+                </el-button>
+            </div>
+        </el-dialog>
+        <dialog-file-component :visible.sync="dialogFileVisible" :group="group" :operation="operation" :type="type"
+            @cancel="cancelFileDialog" @confirm="confirmFileDialog"></dialog-file-component>
     </div>
 </template>
  
 <script>
 import { mapState } from 'vuex'
+import { getGroupList, getStreamOrMeasureList, deleteStreamOrMeasure, deleteGroup, createGroup, editGroup, createFile } from '@/api/index'
+import { Message } from "element-ui"
 import RightMenuComponent from './RightMenuComponent.vue'
+import DialogFileComponent from './DialogFileComponent.vue'
+const list1 = [{
+    icon: "el-icon-folder",
+    name: "new group",
+    id: "create Group"
+}, {
+    icon: "el-icon-folder",
+    name: "edit group",
+    id: "edit Group"
+}, {
+    icon: "el-icon-document",
+    name: "new file",
```

Comment:
The components under a group are the "resources" of the database instead of "file". The conventional way is to show its dedicated name, "stream" or "measure".